### ISSUE TITLE
Camera source is reset when selecting "power on"

### ISSUE THREAD

**Issue by normen** [glikely/obs-ptz#143]
Hi, it seems that for some reason the assigned OBS source of a camera is reset to nothing when I select the "power on" option in the right-click menu.. My RS-485 VISCA cameras do power on but after that the mapping is lost.

**Comment by glikely** [comment]
That is very odd! I'll see if I can reproduce on my end.

Please attach an OBS log file from when the problem occurs to this issue

**Comment by glikely** [comment]
Yup, I see the same behaviour here. I'll dig into it and release a fix.

**Comment by glikely** [comment]
Should be fixed now. I've just pushed out a tag that will create a v0.15.4 release. If all goes well you'll have a binary in about 10 minutes.

**Comment by normen** [comment]
u da man! 🙂
